### PR TITLE
Tehtris: add format

### PR DESCRIPTION
**Pull request by squioc**
Add new format to parse events from Tehtris EDR

### CODE REVIEW

**Review by Darkheir — APPROVED** [review]

File: Tehtris/_meta/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+uuid: 98f11a30-1de6-4619-8ea5-cb000246d5bd
+name: Tehtris
+slug: tehtris
+description: >-
+  [Tehtris](https://tehtris.com) is a cybersecurity company offering products designed to protect endpoints against knwon and unknown threats
```
Comment (otetard) [inline Tehtris/_meta/manifest.yml:5]:
```suggestion
  [Tehtris](https://tehtris.com) is a cybersecurity company offering products designed to protect endpoints against known and unknown threats.
```

Comment (squioc) [reply Tehtris/_meta/manifest.yml:5]: Fixed

File: Tehtris/tehtris-edr/_meta/fields.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+tehtris.edr.host.os.architecture:
+  name: tehtris.edr.host.os.architecture
+  type: keyword
+  description: "The cpu architecture of the operating system"
+
+tehtris.edr.host.is_server:
+  name: tehtris.edr.host.is_server
+  type: boolean
+  description: "Is the host operate as a server"
```
Comment (PierrickV) [inline Tehtris/tehtris-edr/_meta/fields.yml:49]:
```suggestion
  description: "Is the host operating as a server"
```

File: Tehtris/tehtris-edr/_meta/smart-descriptions.json [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    ],
+    "relationships": [
+        {
+            "source": "file.path",
+            "target": "tehtris.edr.file.macro.sha1",
+            "type": "contains malicious macro"
+        }
+    ]
+},{
+    "value": "alert {tehtris.edr.event.type} raised: {event.reason}",
```
Comment (PierrickV) [inline Tehtris/tehtris-edr/_meta/smart-descriptions.json:153]:
```suggestion
    "value": "Alert {tehtris.edr.event.type} raised: {event.reason}",
```

File: Tehtris/tehtris-edr/_meta/fields.yml (hunk 2 above)
Comment (PierrickV) [inline Tehtris/tehtris-edr/_meta/fields.yml:49]:
```suggestion
  description: "Is the host operating as a server"
```

**Review by PierrickV — APPROVED** [review]

File: Tehtris/tehtris-edr/_meta/smart-descriptions.json (hunk 3 above)
Comment (squioc) [reply Tehtris/tehtris-edr/_meta/smart-descriptions.json]: Fixed

**Review by PierrickV — APPROVED** [review]

**Pull request merged**